src/app/query/nonce/account-nonces.query.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery, UseQueryOptions, UseQueryResult } from 'react-query';
+import { AddressNonces } from '@stacks/blockchain-api-client/lib/generated';
+
+import { useCurrentAccountStxAddressState } from '@app/store/accounts/account.hooks';
+import { useCurrentNetworkState } from '@app/store/network/networks.hooks';
+import { useApi } from '@app/store/common/api-clients.hooks';
+
+export function useGetAccountNonces(reactQueryOptions: UseQueryOptions) {
+  const principal = useCurrentAccountStxAddressState();
+  const network = useCurrentNetworkState();
+  const { accountsApi } = useApi();
+
+  const fetchAccountNonces = () => {
+    if (!principal) return;
+    return accountsApi.getAccountNonces({
+      principal,
+    }) as Promise<AddressNonces>;
+  };
+
+  return useQuery({
+    queryKey: ['accountNonces', principal, network],
```

Comment:
Elsewhere these are hyphen cased

Reply:
Ok, yep, this was here but overlooked. Will change it.